src/components/overview/Overview.vue

```diff
@@ -145,6 +142,20 @@ export default {
 			}
 			this.loading = false
 		},
+		sortCards(when) {
+			const cards = this.assignedCardsDashboard[when]
+
+			if (!cards) {
+				return null
+			} else {
+				return cards.toSorted((current, next) => {
+					const currentDueDate = new Date(current.duedate)
+					const nextDueDate = new Date(next.duedate)
+
+					return currentDueDate - nextDueDate
+				})
+			}
+		},
```

Comment:
Just a very small detail. Since the if-branch returns, we don't need an else branch which slightly improves readability:

```suggestion
			return cards.toSorted((current, next) => {
				const currentDueDate = new Date(current.duedate)
				const nextDueDate = new Date(next.duedate)

				return currentDueDate - nextDueDate
			})
		},
```